lib/support/cli.js

```diff
@@ -447,6 +447,11 @@ module.exports.parseCommandLine = function parseCommandLine() {
     .option('resultBaseURL', {
       describe: 'The base URL to the server serving the HTML result. In the format of https://result.sitespeed.io'
     })
+    .option('gzipHAR', {
+      describe: 'Compress the HAR files with GZIP.',
+      default: false,
```

Comment:
True… =(
But we should follow up with gzip support in PerfCascade! And maybe offer "on the fly" decompression if possible (Download as HAR or Download as HAR.gz). Shape the future! =)